app/views/pages/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class='container feature feature-text'>
+    <form method="POST" action="http://formspree.io/kurtzikaras@gmail.com">
```

Comment:
It's fine that you use a plain form here, but you could also convert this to use `form_for`, if you wanted.